src/sidepanel/v2/components/ui/slider.tsx

```
@@ -32,7 +32,7 @@ export function Slider({
     onChange(newValue)
   }
 
-  // Calculate progress percentage for visual feedback
+  // Calculate progress percentage for visual feedback (not rendered)
   const progress = ((value - min) / (max - min)) * 100
```

ESLint

[error] 'progress' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)